src/src/components/form/CustomSection.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <th colspan="4" class="sectionHeader">
+    <h3 v-html="field.title" @click="toggle($event)" :class="{'toggle' : field.canToggleSection}"></h3>
+    <span>
+        <a v-if="field.canDeleteSection"  href=""  @click="removeFields($event)" class="list-btn icon-times"></a>
```

Comment:
Codacy has a fix for the issue: [Require self-closing on HTML elements (<a>).](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
        <a v-if="field.canDeleteSection"  href=""  @click="removeFields($event)" class="list-btn icon-times"/>
```